apps/collabora.sh

```diff
@@ -5,11 +5,7 @@
 # shellcheck disable=2034,2059
 true
 # shellcheck source=lib.sh
```

Comment:
Here's how it should look like. This will fix your travis issues.

Reply:
So shall I add this before every line of the travis error lines? If I remember correctly, than I have already tried that but then a different travis error appeared.

Reply:
I hope that works. will see later on.